src/js/views/globals/patient-modal/patient-modal.hbs

```diff
@@ -2,4 +2,3 @@
 <div class="u-margin--b-8 flex"><div class="modal__form-label">{{ @intl.globals.patientModal.patientModalTemplate.lastName }}</div><div class="flex-grow" data-last-name-region></div></div>
 <div class="u-margin--b-8 flex"><div class="modal__form-label">{{ @intl.globals.patientModal.patientModalTemplate.dateOfBirth }}</div><div data-dob-region></div></div>
 <div class="u-margin--b-8 flex"><div class="modal__form-label">{{ @intl.globals.patientModal.patientModalTemplate.sex }}</div><div data-sex-region></div></div>
-<div class="u-margin--b-8 flex"><div class="modal__form-label">{{ @intl.globals.patientModal.patientModalTemplate.workspaces }}</div><div class="modal__form-component" data-workspaces-region></div></div>
```

Comment:
Realized just now that we'll need this region for the edit patient and view patient modals. And just have it not show if it's the `Add Patient` version of the modal.

So, need to redo some work on this.

Reply:
probably could be `{{#unless isNew}}`.. if we need an empty region for add for the view not to fail could be a blank div in an `{{else}}`

Reply:
I went with just an `{{@unless isNew}}` in the template. And then did a `if (!this.model.isNew()) this.showWorkspacesComponent();` when showing the region.

I don't feel strongly about either way. But it seemed cleaner than doing an empty div 🤷. I'd be fine with either.